bin/grommet.js

```diff
@@ -17,76 +18,120 @@ String.prototype.capitalize = function() {
   return words.join(' ');
 };
 
-function nodeVersionSupported() {
-  return Number(process.version.match(/^v(\d+\.\d+)/)[1]) >= 0.10;
+function fileExists(filePath) {
+  try {
+      result = fs.statSync(filePath);
+      return true;
+  } catch(error) {
+  }
+
+  return false;
+}
+
+function nodeVersionSupported(nodeVersion) {
+  return nodeVersion >= Number(supportedNodeVersion);
 }
 
 function npmVersionSupported(npmVersion) {
-  return npmVersion >= 1.4;
+  return npmVersion >= Number(supportedNpmVersion);
 }
 
-var tasks = {
-  'init': function(done) {
-    if (!nodeVersionSupported() || !npmVersionSupported(npmVersion)) {
-      console.error('[grommet] Grommet requires Node v0.10+ and NPM 1.4.x+.');
-      console.error(
-        '[grommet] Currently you have Node ' + process.version +
-        ' and NPM ' + npmVersion
-      );
-      process.exit(1);
-    }
+function dependenciesSupported(nodeVersion, npmVersion) {
+  if (!nodeVersionSupported(nodeVersion) || !npmVersionSupported(npmVersion)) {
+    console.error(
+      '[grommet] Grommet requires Node v'+supportedNodeVersion+
```

Comment:
can we add space in here? as you have for the other console messages?

'[grommet] Grommet requires Node v' + supportedNodeVersion +